modules/data_team_helper/README.md

```diff
@@ -0,0 +1,29 @@
+# Data Team Helper
+
+## Purpose
+
+The Data Team Helper module is intended to provide a dashboard to
```

Comment:
maybe we should mention somewhere that the module is called 'Quality Control' under tools menu instead of Data Team Helper

Reply:
sorry for the delay.. I just added it under "Integrations with LORIS", because that seemed like the most logical place.